accounts-db/src/accounts.rs

```diff
@@ -562,51 +562,60 @@ impl Accounts {
         &self,
         txs: impl Iterator<Item = &'a Tx>,
         tx_account_lock_limit: usize,
+        relax_intrabatch_account_locks: bool,
     ) -> Vec<Result<()>> {
         // Validate the account locks, then get iterator if successful validation.
-        let tx_account_locks_results: Vec<Result<_>> = txs
```

Comment:
`Accounts::lock_accounts()` could be reimpled as a wrapper on `lock_accounts_with_results()` or possibly deleted, it isnt really required anymore since all batch-building needs to have results